docs/component/identity-wallet.md

```diff
@@ -4,8 +4,28 @@ title: Identity Mobile App
 sidebar_label: Identity Mobile App
 ---
 
-Overview
+OA provides it's own mobile application to view, store and share Verifiable Document and Transferable Records.
 
-## Header
+## Installation
+
+For the moment, the application is not available on Apple store or Android store. You will need to run it through [Expo](https://expo.io/).
+
+1. Download Expo [![Expo on apple store](/docs/component/identity-wallet/apple-store.svg)](https://apps.apple.com/fr/app/expo-client/id982107779 "Open expo on apple store") [![Expo on apple store](/docs/component/identity-wallet/android-store.svg "Open expo on android store")](https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en)
+1. Open Expo and download Identity Wallet app by scanning the following QR Code
+
+<img src="/docs/component/identity-wallet/identity-wallet-qr-code.png" alt="identity wallet qr code" title="scan the qr code with expo" width="200"/>
+
+🎉 Congratulations, you are now able to use Identity Wallet application.
+
+## Documents
+
+In order to view a document, open Identity Wallet, proceed to the QR Code tab and scan. Some examples are available on [Open Attestation Gallery](http://gallery.openattestation.com/).
+
+In the gallery you will find 2 different kinds of document, viewable and storable:
+- Viewable documents are documents that you can only view in the application. No other action is permitted.
```

Comment:
don't need documents again. just "Viewable documents are that you can only view in the application"

Reply:
Is there a missing word ? Doesn't sound valid to me.

I think repetition is fine in this case

Reply:
In the gallery you will find 2 different kinds of documents, viewable and storable:
- Viewable documents can only be viewed (and not stored) in the application. No other actions are permitted.
- Storable documents can be viewed and stored in the application. Once a document is stored into your wallet you can also share it with others. When you view the document, a QR Code icon will be displayed. Click on it to share your document.

Reply:
changed